timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/HealthStatus.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.adjudicate;
+
+import java.util.HashSet;
+
+import com.google.common.collect.Sets;
+
+public enum HealthStatus {
```

Comment:
Like the internal proxy we have for checking whether our service is well-behaved, it is probably a good idea to include in the UNHEALTHY case some information about why we think things are unhealthy.

Reply:
we ended up adding this at a different layer, but this information is still reported, so we should be good here.